app/src/main/java/com/example/nonameapp/base/BaseActivity.kt

```diff
@@ -5,7 +5,7 @@ import android.view.LayoutInflater
 import androidx.appcompat.app.AppCompatActivity
 import androidx.viewbinding.ViewBinding
 
-abstract class BaseActivity<VB : ViewBinding>(
+abstract class BaseActivity<VB : ViewBinding, LoginFragment>(
```

Comment:
Sao lại có LoginFragment ở đây? Không thay đổi base nhé
```suggestion
abstract class BaseActivity<VB : ViewBinding>(
```